src/server_manager/web_app/app.spec.ts

```diff
@@ -35,41 +32,43 @@ const TOKEN_WITH_ONE_SERVER = 'one-server-token';
 // tslint:disable-next-line:no-any
 (global as any).bringToFront = () => {};
 
+// Inject app-root element into DOM once before the test suite runs
+beforeAll(async () => {
+  const loadAppRoot = new AppRoot();  // TODO: Remove once figure out how to load AppRoot properly.
```

Comment:
Just for context, eight tests fail with the the following error when I remove this line:

```
Chrome Headless 85.0.4183.121 (Mac OS 10.15.6) App shows progress screen when starting with DigitalOcean servers still being created FAILED
        TypeError: this.appRoot.showIntro is not a function
            at App.showIntro (webpack:///./src/server_manager/web_app/app.ts?:625:22)
            at App.eval (webpack:///./src/server_manager/web_app/app.ts?:310:18)
            at <Jasmine>
            at eval (webpack:///./src/server_manager/web_app/app.ts?:34:71)
            at <Jasmine>
            at __awaiter (webpack:///./src/server_manager/web_app/app.ts?:30:12)
            at App.start (webpack:///./src/server_manager/web_app/app.ts?:309:16)
            at eval (webpack:///./src/server_manager/web_app/app.spec.ts?:175:19)
            at <Jasmine>
            at fulfilled (webpack:///./src/server_manager/web_app/app.spec.ts?:22:58)
```

Reply:
It looks like the constructor initialization is needed. Could you add a comment explaining?

Reply:
Done.